lib/monthly.rb

```diff
@@ -0,0 +1,120 @@
+# frozen_string_literal: true
+
+module Declarations
+  module Prefills
+    module Prefillables
+      class MonthlyVatReturnService < ApplicationService
+        SENTRY_OWNER = Constants::SentryOwners::DECLARATIONS
+
+        private
+
+        IMPORTATION_VAT_RATES = [0.2, 0.1, 0.085, 0.055, 0.021, 0.0105].freeze
+
+        def initialize(declaration:, prefill:)
+          @declaration = declaration
+          @declaration_without_prefills = @declaration.dup
```

Comment:
Avoid duplicating the `@declaration` object inside the `initialize` method. If you need to compare the original state, consider other patterns such as the Memento pattern or explicitly passing a previous state if necessary.